gotocompany/shield/v1beta1/shield.proto

```diff
@@ -584,6 +584,8 @@ message ListUsersRequest {
   int32 page_size = 1;
   int32 page_num = 2;
   string keyword = 3;
+  string sort = 4;
+  string order = 5;
```

Comment:
it's a bit hard to determine which one is for column name which one for order direction

`order_by` (for column name_ and `sort` (asc | desc) would be better I guess

Reply:
no I think the `sort` could be kept for column name as sort by.
just the direction could be configured to be clearer. probably we could call it `direction` like how compass does.
wdyt?

Reply:
`direction` or `sort_direction` sounds good to me

Reply:
updated to `direction`